core/src/main/resources/hudson/model/Computer/_api.jelly

```diff
@@ -43,5 +43,14 @@ THE SOFTWARE.
       Or you can POST to <a href="../changeOfflineCause?offlineMessage=some%20text">this URL</a>
       to mark this node temporary offline, or adjust the offline cause message if it is already offline.
   </p>
+    <h2>Retrieving Agent Secret</h2>
+    <p>
+        To retrieve the agent secret (JNLP MAC) via the API, you can access <a href="../agent-secret/">this URL</a>.
+        This requires the Computer.CONNECT permission and returns the plain text secret that can be used for agent authentication.
+    </p>
+    <p>
+        If you need to retrieve the secret for a specific agent from a script, you can access:
+        <code>JENKINS_URL/computer/AGENT_NAME/agent-secret/</code>
```

Comment:
Redundant with the previous paragraph considering right-click, "Copy link URL"? Or what does this add?